src/main/java/com/checkout/payments/four/request/source/ProviderTokenSource.java

```diff
@@ -0,0 +1,40 @@
+package com.checkout.payments.four.request.source;
+
+import com.checkout.common.PaymentSourceType;
+import com.checkout.marketplace.MarketplaceAccountHolder;
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public final class ProviderTokenSource extends AbstractRequestSource {
+
+    @SerializedName("payment_method")
+    private String payment_method;
+
+    private String token;
+
+    @SerializedName("account_holder")
+    private MarketplaceAccountHolder accountHolder;
```

Comment:
Classes from another modules should not be imported. Please use the ones in common.

In this case, what do you think in moving `AccountHolder` to `common` (instead of `common.four`) and use that one? This is the first scenario were there's a CS1 AccountHolder reference to the generic one (like it's been happening with Four schemas), so I believe it's the right move.

Reply:
EDIT: use `common.four.AccountHolder`.